toolchain/lower/file_context.h

```diff
@@ -47,6 +47,25 @@ class FileContext {
     return functions_[function_id.index];
   }
 
+  // Gets a or creates callable's function. Returns nullptr for a builtin.
+  auto GetOrCreateFunction(SemIR::FunctionId function_id,
+                           SemIR::SpecificId specific_id) -> llvm::Function* {
```

Comment:
This seems long enough that maybe it belongs in the cpp

Reply:
Done.